src/pages/ThorChainLP/components/AddLiquitity/AddLiquidityInput.tsx

```diff
@@ -284,43 +342,80 @@ export const AddLiquidityInput: React.FC<AddLiquidityProps> = ({
     )
   }, [asset, foundPool, rune, translate])
 
+  const buyAssetSearch = useModal('sellAssetSearch')
```

Comment:
```suggestion
  const buyAssetSearch = useModal('buyAssetSearch')
```

Reply:
Fixed in https://github.com/shapeshift/web/pull/6028/commits/d943ba2003513a86a8d7066ad82213822a9db4e5

Reply:
Haha, I had actually used that one initiallly https://github.com/shapeshift/web/pull/6028#discussion_r1455487538 but decided sell made more sense!